grovedb/src/operations/proof/mod.rs

```diff
@@ -0,0 +1,154 @@
+//! Proof operations
+
+#[cfg(feature = "full")]
+mod generate;
+pub mod util;
+mod verify;
+
+use std::{collections::BTreeMap, fmt};
+
+use bincode::{Decode, Encode};
+use derive_more::From;
+use grovedb_merk::proofs::{query::Key, Decoder, Node, Op};
+
+use crate::operations::proof::util::{element_hex_to_ascii, hex_to_ascii};
+
+#[derive(Debug, Clone, Copy, Encode, Decode)]
+pub struct ProveOptions {
+    pub decrease_limit_on_empty_sub_query_result: bool,
```

Comment:
Please add a more descriptive comment on the purpose of this field. Since it it passed on the proof APIs

Reply:
(Someone has to play the role of @thephez)